.github/workflows/build.yaml

```
@@ -0,0 +1,158 @@
+    runs-on: ubuntu-latest
+    outputs:
+      PH_EE_CONNECTOR_MPESA_IMAGE_TAG: ${{ steps.set-image-tag.outputs.IMAGE_TAG }}
+    steps:
+    - name: Check out source code
+      uses: actions/checkout@v4.2.2
+      with:
+        fetch-depth: 0
+        fetch-tags: true
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
```

actionlint (1.7.7)

the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)